File: 프로그래머스/unrated/181878. 원하는 문자열 찾기/원하는 문자열 찾기.py
def solution(myString, pat):
    # myString과 pat 모두 대문자로 바꾼 뒤, in 연산자로 포함 여부를 가린다.
    myString = myString.upper()
    pat = pat.upper()
    
    return 1 if pat in myString else 0

File: 프로그래머스/unrated/181878. 원하는 문자열 찾기/원하는 문자열 찾기.js
function solution(myString, pat) {
    // myString, pat을 모두 대문자로 바꾼 뒤, includes 연산자로 포함 여부를 가린다.
    return myString.toUpperCase().includes(pat.toUpperCase()) ? 1 : 0
}